tests/hacker.test.js

```diff
@@ -12,6 +12,29 @@ const util = {
 };
 
 const storedHacker1 = util.hacker.HackerA;
+const newHacker1 = util.hacker.newHacker1;
+
+describe("POST create hacker", function () {
+    it("should SUCCEED and create a new hacker", function(done) {
+        chai.request(server.app)
+        .post(`/api/hacker/`)
+        .type("application/json")
+        .send(newHacker1)
+        .end(function (err, res) {
+            res.should.have.status(200);
+            res.should.be.json;
+            res.body.should.have.property("message");
+            res.body.message.should.equal("Hacker creation successful");
+            res.body.should.have.property("data");
+
+            // delete _id and status because those fields were generated
+            delete res.body.data._id;
+            delete res.body.data.status;
+            chai.assert.equal(JSON.stringify(res.body.data), JSON.stringify(newHacker1));
+            done();
+        });
+    });
+});
```

Comment:
Can you make a test to verify that invalid input data is properly failed?

Reply:
Done